openpype/plugins/load/delivery.py

```diff
@@ -88,6 +88,11 @@ def __init__(self, contexts, log=None, parent=None):
         template_label.setCursor(QtGui.QCursor(QtCore.Qt.IBeamCursor))
         template_label.setTextInteractionFlags(QtCore.Qt.TextSelectableByMouse)
 
+        renumber_frame = QtWidgets.QCheckBox()
+
+        renumber_frame_value = QtWidgets.QSpinBox()
+        renumber_frame_value.setMinimum(-(1 << 32) // 2)
```

Comment:
We need to set max value to, otherwise it allows only `99`

```suggestion
        # set minimum and maximum values with the size of 32bit integer.
        max_int = (1 << 32) // 2
        renumber_frame_value.setRange(-max_int, max_int - 1)
```